app/src/main/res/values/strings.xml

```diff
@@ -47,7 +47,7 @@
     <string name="message_ctcp_version">Received a CTCP version</string>
 
     <string name="command_error_not_found">No such command was found.</string>
-    <string name="command_error_internal">An internal error occurred when trying to process the command.</string>
+    <string name="command_error_internal">Could not process the command.</string>
```

Comment:
I do not agree with the changes replacing "An internal error has occurred during *" with "Could not *". This is because the internal errors were meant as a "catch all" sort of error and preferably replaced in the future with more detailed messages, and the "internal error" wording is preferred to me.